client/benches/torii.rs

```diff
@@ -150,15 +143,18 @@ fn instruction_submits(criterion: &mut Criterion) {
             .domain("wonderland".parse().expect("Valid"))
             .account(
                 "alice".parse().expect("Valid"),
-                configuration.public_key.clone(),
+                configuration.common.key_pair.public_key().clone(),
```

Comment:
```suggestion
                configuration.key_pair.public_key().clone(),
```

IMO, this is not a good API